### PR TITLE
v2021.11.9

### PR DESCRIPTION
**Pull request by amsryq**

### CODE REVIEW

File: src/CrowdedMod/CrowdedModPlugin.cs [hunk 1]
```diff
@@ -4,23 +4,20 @@
...
-namespace CrowdedMod {
-    [BepInPlugin(Id)]
-    [BepInProcess("Among Us.exe")]
-    [BepInDependency(ReactorPlugin.Id)]
-    public class CrowdedModPlugin : BasePlugin
-    {
-        public const string Id = "pl.przebor.crowded";
+namespace CrowdedMod;
+
+[BepInAutoPlugin("pl.przebor.crowded")]
```
Comment (Galster-dev) [inline src/CrowdedMod/CrowdedModPlugin.cs]: @przebor why?

File: src/CrowdedMod/Patches/CreateGameOptionsPatches.cs [hunk 2]
```diff
@@ -1,149 +1,150 @@
...
+                    __instance.SetMaxPlayersButtons(value);
+                }));
+                        
+                button.transform.position += new Vector3(offset, 0, 0);
+            });
+    
+            playerButtons.Insert(0, minusButton);
+            playerButtons.Add(plusButton);
+    
+            var ppp = new Il2CppSystem.Collections.Generic.List<SpriteRenderer>();
```
Comment (Galster-dev) [inline src/CrowdedMod/Patches/CreateGameOptionsPatches.cs:95]: What is this naming?

File: src/CrowdedMod/Patches/CreateGameOptionsPatches.cs [hunk 3]
```diff
@@ -1,149 +1,150 @@
...
+            foreach (var pl in playerButtons) ppp.Add(pl);
```
Comment (Galster-dev) [inline src/CrowdedMod/Patches/CreateGameOptionsPatches.cs:96]: I'm not sure, but I think there is a better way to cast

**Review by Galster-dev — COMMENTED** [review]
Quick code review
Requesting @przebor logic review

File: src/CrowdedMod/CrowdedModPlugin.cs (hunk 1 above)
Comment (przebor) [reply src/CrowdedMod/CrowdedModPlugin.cs]: Do you mean `pl.przebor.crowded`?

Comment (przebor) [reply src/CrowdedMod/CrowdedModPlugin.cs]:
```suggestion
[BepInAutoPlugin("xyz.crowdedmods.crowdedmod")]
```

File: src/CrowdedMod/Patches/CreateGameOptionsPatches.cs (hunk 2 above)
Comment (amsryq) [reply src/CrowdedMod/Patches/CreateGameOptionsPatches.cs:95]: Sorry, that was meant to be temporary

**Comment by Galster-dev** [comment]
Sorry I'm kinda late. Been waiting for @przebor review

Looks like we don't need this anymore, but thank you for your contribution!

**Pull request closed without merging**